listenbrainz/db/model/playlist.py

```diff
@@ -152,7 +151,7 @@ def serialize_jspf(self):
 
         tracks = []
         for rec in self.recordings:
-            tr = {"identifier": PLAYLIST_TRACK_URI_PREFIX + str(rec.mbid)}
+            tr = {"identifier": [PLAYLIST_TRACK_URI_PREFIX + str(rec.mbid)]}
```

Comment:
Doesn't the JSPF spec mandate this to be a single identifier? I suggest using an additional_identifiers or mbids or some one well named field in the extension to have multiple MBIDs.

Reply:
Good catch -- multiple IDs are not really needed, so easy fix.